thunder/tests/test_core.py

```diff
@@ -1620,6 +1620,19 @@ def bar(__a, __b):
     thunder.trace()(bar, a, b)
 
 
+# Tests if Thunder can trace a function without a valid signature
+@instantiate(dtypes=NOTHING)
```

Comment:
Q: with this setting, how many instances are created? or how many instances would you think are needed?

Reply:
To clarify, I think the necessity level to test this three times is the same as its adjacent tests.